extensions/industry/healthcare/fhir/datatypes/range.schema.json

```diff
@@ -0,0 +1,35 @@
+{
+      "meta:license":[
+      "Copyright 2024 Adobe Systems Incorporated. All rights reserved.",
+      "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+      "you may not use this file except in compliance with the License. You may obtain a copy",
+      "of the License at https://creativecommons.org/licenses/by/4.0/"
+   ],
+   "$schema":"http://json-schema.org/draft-06/schema#",
+   "$id":"https://ns.fhir.org/datatypes/range",
+   "title":"Range",
+   "type":"object",
+   "description":"Set of values bounded by low and high.(This datatype is created as per HL7 FHIR Release 5 specifications)",
+   "definitions":{
+      "range":{
+         "properties":{
+            "xdm:low":{
+               "title":"Low",
+               "description":"Low Limit",
+               "type":"string"
```

Comment:
As per FHIR the type is SimpleQuantity (which is like Quantity datatype without "comparator"). So we will need to define that datatype

Reply:
I was thinking the same but unfortunately there is not structure defined for [SimpleQuantity ](https://hl7.org/fhir/datatypes.html#SimpleQuantity). I will dig into data type usage examples to figure out the structure.